lib/config.js

```diff
@@ -351,7 +351,26 @@ const CONFIG_SYNTAX_HELP = '  module.exports = function(config) {\n' +
   '    });\n' +
   '  };\n'
 
-function parseConfig (configFilePath, cliOptions) {
+function parseConfig (configFilePath, cliOptions, parseOptions) {
+  function fail () {
+    log.error(...arguments)
```

Comment:
Is this supported on IE?

Reply:
This is server-side code, so IE is not relevant here.